### PR TITLE
tests: on_target: update readme

### PR DESCRIPTION
**Pull request by DematteisGiacomo**
Readme needs an update due to latest changes.

### CODE REVIEW

File: tests/on_target/README.md [hunk 1]
```diff
@@ -16,59 +15,64 @@ docker run --rm -it \
...
+Additional fota envs
 ```shell
-export SEGGER=<your_segger>
 export IMEI=<your_imei>
 export FINGERPRINT=<your_fingerprint>
-pytest -s -v -m "dut1 and fota" tests
 ```
 
-### Run DFU tests
+Run desired tests
```
Comment (dilner) [inline tests/on_target/README.md:42]: Specify that this isn't all tests but just examples.

File: tests/on_target/README.md [hunk 2]
```diff
@@ -16,59 +15,64 @@ docker run --rm -it \
...
 
-### Run FOTA tests
-
-Precondition: thingy91x with segger fw on 53
-
+Additional fota envs
 ```shell
-export SEGGER=<your_segger>
 export IMEI=<your_imei>
 export FINGERPRINT=<your_fingerprint>
```
Comment (dilner) [inline tests/on_target/README.md:39]: In order to run memfault tests we need to se those variables as well.

File: tests/on_target/README.md [hunk 3]
```diff
@@ -2,12 +2,11 @@
 
 ## Run test locally
 
+NOTE: The tests have been tested on Ubuntu 22.04. For details on how to install Docker please refer to the Docker documentation https://docs.docker.com/engine/install/ubuntu/
+
 ### Setup docker
 ```shell
 docker pull ghcr.io/hello-nrfcloud/firmware:docker-v1.0.3
-cd <path_to_oob_dir>
```
Comment (dilner) [inline tests/on_target/README.md:8]: shouldn't we specify where to run this?

File: tests/on_target/README.md [hunk 4]
```diff
@@ -16,59 +15,64 @@ docker run --rm -it \
...
-```shell
-nrfutil device list
-```
-
-### Run UART tests
-
+### NRF91 tests
 Precondition: thingy91x with segger fw on 53
 
+Get device id and set env
```
Comment (dilner) [inline tests/on_target/README.md:30]: I think we should have to separate shell command here, "get device id" and "set environment variables"

File: tests/on_target/README.md [hunk 5]
```diff
@@ -16,59 +15,64 @@ docker run --rm -it \
...
 Precondition: thingy91x with external debugger attached
 
 IMPORTANT: switch must be on nrf53.
 
-Set all this bunch of env variables appropriately (see worflow for details):
-SEGGER_NRF53, SEGGER_NRF91, UART_ID, NRF53_HEX_FILE, NRF53_APP_UPDATE_ZIP,
-NRF53_BL_UPDATE_ZIP, NRF91_HEX_FILE, NRF91_APP_UPDATE_ZIP, NRF91_BL_UPDATE_ZIP
+Set env
+```shell
+export SEGGER_NRF53=<>
```
Comment (dilner) [inline tests/on_target/README.md:58]: <your_>

File: tests/on_target/README.md [hunk 6]
```diff
@@ -16,59 +15,64 @@ docker run --rm -it \
...
+pytest -s -v -m "slow" tests/test_bridge/test_conn_bridge.py::test_conn_bridge
+```
+
+### PPK test
+
+Precondition: thingy91x with segger fw on 53 and PPK setup
+
+Set env
+```shell
+export SEGGER_PPK=<>
```
Comment (dilner) [inline tests/on_target/README.md:74]: <your_>

File: tests/on_target/README.md (hunk 3 above)
Comment (DematteisGiacomo) [reply tests/on_target/README.md:8]: yes right

**Pull request merged**